SmartDeviceLink/SDLPreloadChoicesOperation.m

```diff
@@ -118,9 +118,18 @@ - (void)sdl_preloadCells {
 
     NSMutableArray<SDLCreateInteractionChoiceSet *> *choiceRPCs = [NSMutableArray arrayWithCapacity:self.cellsToUpload.count];
     for (SDLChoiceCell *cell in self.cellsToUpload) {
-        [choiceRPCs addObject:[self sdl_choiceFromCell:cell]];
+        SDLCreateInteractionChoiceSet *csCell =  [self sdl_choiceFromCell:cell];
+        if(csCell != nil) {
+            [choiceRPCs addObject:csCell];
+        }
     }
-
+    
+    if (choiceRPCs.count == 0) {
+         SDLLogE(@"Error choiceRPCs is an empty array");
```

Comment:
This error needs to be useful to developers, and they won't know what `choiceRPCs` is. This should be something like "All choice cells to send are `nil`, so the choice set will not be shown."

Reply:
Okay will fix